guides/release/routing/specifying-a-routes-model.md

```diff
@@ -224,3 +343,14 @@ export default Route.extend({
 ```
 
 And calling `modelFor` returned the result of the `model` hook.
+
+## Debugging models
+
+If you are having trouble getting a model's data to show up in the template, here are some tips:
+
+- return static, sample data as a test to see if the problem is really in the model hook, or elsewhere down the line
+- study JavaScript Promises in general, to make sure you are returning data from the Promise correctly
+- make sure your `model` hook has a `return`
+- check to see whether the data returned from a `model` hook is an object, array, or JavaScript Primitive. For example, if the results of `model` are an array, using `{{this.model}}` in the template won't work. You will need to iterate over the results with an `{{#each}}` helper. If the results of `model` are a single object, you need to access the individual attributes like `{{this.model.title}}` to see anything rendered
```

Comment:
Mix of singular and plural in this sentence. The `model` hook returns one result, and it can be an object / array / string etc.  Suggested changes:

For example, if the **result** of `model` is an array, using `{{this.model}}` in the template won't work. You will need to iterate over the **array** with an `{{#each}}` helper. If the result **is an object**, you need to access the individual **attribute** like `{{this.model.title}}` to **render it in the template.**